dockers/tools/cli/loadtest/Dockerfile

```diff
@@ -67,7 +67,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN GO_VERSION="$(< GO_VERSION)" \
     && VALD_VERSION="$(< VALD_VERSION)" \
     && GIT_COMMIT="$(git rev-list -1 HEAD)" \
-    && CPU_INFO_FLAGS="$(cat /proc/cpuinfo | grep flags | cut -d " " -f 2- | head -1)" \
+    && CPU_INFO_FLAGS="$(< /proc/cpuinfo | grep flags | cut -d " " -f 2- | head -1 || true)" \
```

Comment:
why do we need `|| true` here?

Reply:
I fixed it because with set -o pipefail, if you run a head or grep command on the right side of the pipe, the pipe will be closed and a write to the closed pipe will occur, resulting in a SIGPIPE.